### PR TITLE
Updated doctrine/coding-standard to 6.0

### PR DESCRIPTION
**Pull request by patrickjahns**
Updated coding-standard and used `phpcbf` to automatically fix them

### CODE REVIEW

File: lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php [hunk 1]
```diff
@@ -103,6 +103,7 @@ public static function sortByField($name, $orientation = 1, ?Closure $next = nul
 
         return static function ($a, $b) use ($name, $next, $orientation) : int {
             $aValue = ClosureExpressionVisitor::getObjectFieldValue($a, $name);
+
```
Comment (Ocramius) [inline lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php]: This seems incorrect

Comment (patrickjahns) [reply lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php]: If I revert the change - codesniffer screams:

```
FILE: lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 104 | ERROR | [x] Expected 1 lines after "return", found 0.
     |       |     (SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Need to have a closer look  into the slevomat rules to figure out whats going wrong here

Comment (Ocramius) [reply lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php]: Could you reproduce this and report it to `doctrine/coding-standard` maybe? We can roll with this patch meanwhile 👍

Comment (patrickjahns) [reply lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php]: Boiled it down to a simple example and open report here: https://github.com/slevomat/coding-standard/issues/635

**Pull request merged**